advanced/sr_gui_bootloader/src/sr_gui_bootloader/bootloader.py

```diff
@@ -40,21 +37,22 @@ def __init__(self, parent, nb_motors_to_program, prefix):
         self.parent = parent
         self.nb_motors_to_program = nb_motors_to_program
         self.prefix = prefix
+        self.bootloader_service = None
 
     def run(self):
         """
         perform bootloading on the selected motors
         """
         bootloaded_motors = 0
-        firmware_path = self.parent._widget.txt_path.text()
+        firmware_path = self.parent._widget.txt_path.text()  # pylint: disable=W0212
         for motor in self.parent.motors:
             if motor.checkbox.checkState() == Qt.Checked:
                 try:
                     self.bootloader_service = rospy.ServiceProxy(self.prefix + '/SimpleMotorFlasher',
                                                                  SimpleMotorFlasher)
                     resp = self.bootloader_service(firmware_path, motor.motor_index)
-                except rospy.ServiceException as e:
-                    self.failed.emit("Service did not process request: %s" % str(e))
+                except rospy.ServiceException as exception:
+                    self.failed.emit("Service did not process request: %s" % str(exception))
```

Comment:
Can we please try to use f-strings? --> f"Service did not process request: {exception}"